recipe/meta.yaml

```diff
@@ -76,15 +85,15 @@ outputs:
 
   - name: matplotlib
     build:
-      skip: true  # [py<37]
+      skip: true  # [py<38]
     requirements:
       host:
         - python
       run:
         - python
-        - pyqt  # [(linux64 or win) and python_impl == "cpython"]
-        - tornado
-        - {{ pin_subpackage('matplotlib-base', max_pin="x.x.x.x") }}
+        - pyqt  >=5  # [(linux64 or win) and python_impl == "cpython"]
+        - tornado >=5
```

Comment:
I can't find this requirement upstream. Can you provide a link as a comment so I can verify this?

Reply:
The optional dependencies are listed here: https://matplotlib.org/stable/devel/dependencies.html

Reply:
Thank you!